arlo-client/src/components/Audit/RoundManagement/RoundProgress.tsx

```diff
@@ -1,8 +1,60 @@
 import React from 'react'
+import { ProgressBar } from '@blueprintjs/core'
+import styled from 'styled-components'
 import H2Title from '../../Atoms/H2Title'
+import { IBallot, IAuditBoard } from '../../../types'
+import { IRound } from '../useRoundsJurisdictionAdmin'
 
-const RoundProgress = () => {
-  return <H2Title>Round 1 Progress</H2Title>
+const MainBarWrapper = styled.div`
+  margin-bottom: 20px;
+  width: 500px;
+  font-size: 16px;
+`
+
+const SmallBarWrapper = styled.div`
+  margin-bottom: 10px;
+  width: 400px;
+`
+
+const RoundProgress = ({
+  rounds,
+  ballots,
+  auditBoards,
+}: {
+  rounds: IRound[]
+  ballots: IBallot[]
+  auditBoards: IAuditBoard[]
+}) => {
+  const completeBallots = ballots.filter(b => b.status).length
+  const ballotsPerAB = auditBoards.map(a => {
+    const all = ballots.filter(b => b.auditBoard!.id === a.id)
+    return [all.filter(b => b.status).length, all.length]
+  })
```

Comment:
these counts are already available on the `auditBoard` object at `auditBoard.currentRoundStatus.numSampledBallots` and `auditBoard.currentRoundStatus.numAuditedBallots` 😸

Reply:
Which means, if you'd like, the large status bar progress can be computed just by summing across the audit board stats - you don't even need the list of ballots

Reply:
@jonahkagan Huh, must have missed that. It's not in the types anywhere on this branch. I see a `currentRoundStatus` on the `jurisdictions` object, though it has `numBallotsSampled` and `numBallotsAudited` which are slightly different. Could you get me the full type for what's currently returned from `/election/:electionId/jurisdiction/:jurisdictionId/round/:roundId/audit-board`?

Reply:
@jonahkagan Okay, rebased on #480 and cleaned this up a bit. Can you double check that I'm using `useAuditBoards` correctly?